system/src/system_cloud_internal.cpp

```diff
@@ -886,13 +886,13 @@ void Spark_Protocol_Init(void)
 
 int Send_Firmware_Update_Flags()
 {
-    if (!System.updatesEnabled()) {
+    if (true || !System.updatesEnabled()) {
     	// force the event to be resent. The cloud assumes updates
     	// are enabled by default.
     	system_refresh_flag(SYSTEM_FLAG_OTA_UPDATE_ENABLED);
     }
 
-    if (System.updatesForced()) {
+    if (true || System.updatesForced()) {
     	system_refresh_flag(SYSTEM_FLAG_OTA_UPDATE_FORCED);
```

Comment:
@m-mcgowan could you please add some comments in the code to help explain why these are short circuited `true` and if they will ever be changed in the future?  Basically a summary of what's stated in the PR would be good to capture in the code here.

Reply:
Great point, done!